### ISSUE TITLE
The data returned is inconsistent!

### ISSUE THREAD

**Issue by eecjimmy** [INFURA/infura#155]
![t](https://user-images.githubusercontent.com/7401963/48056038-e926d300-e1eb-11e8-8a63-6a07edd816f8.png)
526和527请求的数据是相同的, 但是返回的数据却是不一致的.

**Comment by hhppkjhb** [comment]
Yes, this will happen when you use Infura. The load balancer that they use will route your requests to different nodes, which seems to be what happened in the above situation. (One of the nodes that served your request is out of sync) Our team experienced similar inconsistency issues and solved it by switching node providers. We currently use [Alchemy](https://alchemyapi.io/) and haven't had issues since.
